localization/en_US.inc

```diff
@@ -1,8 +1,8 @@
 <?php
 $labels = array();
-$labels['automaticallycollected']  = 'Automatically Collected';
-$labels['useautoabook']  = 'Use automatic address book';
-$labels['useforcompletion'] = 'Use automatically collected addresses for autocompletion';
-$labels['contactremoved'] = 'Contact(s) deleted from automatic address book';
-$labels['contactmoved'] = 'Contact(s) moved from automatic address book';
+$labels['automaticallycollected']  = 'Автоматично събрани адреси';
```

Comment:
please don't include bulgarian translations in en_US file. move them in bg_BG instead.